### PR TITLE
abbreviate_artistsort: merge 1.0 branch debug cleanup by @mineo into 2.0

### PR DESCRIPTION
**Pull request by hoxia**
Merges two commits (0b36600c6526208c51848119442f99e06960035d, 2bd784f76d4e301cfcfcad7dbe97c735265a5a2c) by @mineo made against 1.0 branch into 2.0. These commits clean up debug code, and reportedly fix an error which renders the plugin nonfunctional. This merge was suggested in #166, however I have not tested it and the changes should be reviewed with particular attention to the API bump.

I've also incremented the plugin version to reflect the changes.

### CODE REVIEW

**Review by hoxia — COMMENTED** [review]
Oops, referenced the wrong commits in the description. Meant "(0b36600c6526208c51848119442f99e06960035d, 2bd784f76d4e301cfcfcad7dbe97c735265a5a2c)".

File: plugins/abbreviate_artistsort/abbreviate_artistsort.py [hunk 1]
```diff
@@ -90,11 +88,9 @@ def abbreviate_artistsort(tagger, metadata, track, release):
         unsorts = list(metadata.getall(unsortTag))
         for i in range(0, min(len(sorts), len(unsorts))):
             sort = sorts[i]
-            if _debug_level > 1:
-                print("%s: Trying to abbreviate '%s'." % (PLUGIN_NAME, sort))
+            log.debug("%s: Trying to abbreviate '%s'." % (PLUGIN_NAME, sort))
```
Comment (zas) [inline plugins/abbreviate_artistsort/abbreviate_artistsort.py:91]: Note that you can use the following syntax:

```python
log.debug("%s: Trying to abbreviate '%s'.", PLUGIN_NAME, sort)
```

File: plugins/abbreviate_artistsort/abbreviate_artistsort.py [hunk 2]
```diff
@@ -90,11 +88,9 @@ def abbreviate_artistsort(tagger, metadata, track, release):
...
             if sort in _abbreviate_cache:
-                if _debug_level > 3:
-                    print("  Using abbreviation found in cache: '%s'." % (_abbreviate_cache[sort]))
+                log.debug("  Using abbreviation found in cache: '%s'." % (_abbreviate_cache[sort]))
```
Comment (zas) [inline plugins/abbreviate_artistsort/abbreviate_artistsort.py:93]: Is the whitespace at the start of any use?

File: plugins/abbreviate_artistsort/abbreviate_artistsort.py [hunk 3]
```diff
@@ -228,13 +217,9 @@ def abbreviate_artistsort(tagger, metadata, track, release):
                         unsorts[i],
                         new_unsort,
                     )
-                    if _debug_level > 0:
-                        print()
-                        print("Error: Unsorted text for %s has changed from '%s' to '%s'!" % (unsortTag, unsorts[i], new_unsort))
-                        print()
+                    log.warning("Error: Unsorted text for %s has changed from '%s' to '%s'!" % (unsortTag, unsorts[i], new_unsort))
```
Comment (zas) [inline plugins/abbreviate_artistsort/abbreviate_artistsort.py:220]: That's a warning or an error ?

**Review by phw — APPROVED** [review]

**Pull request merged**